packages/ui/src/test/index.test.ts

```diff
@@ -0,0 +1,20 @@
+import { join, dirname, basename } from 'node:path';
+import { readFile } from 'node:fs/promises';
+import glob from 'fast-glob';
+
+describe( 'index', () => {
+	it( 'should export something from each component', async () => {
+		// As described in the CONTRIBUTING.md file, each component should be
+		// exported from an index.ts in its implementation directory.
+		const components = await glob( join( __dirname, '../*/index.ts' ) );
```

Comment:
I'm paranoid about breaking in Windows again, could this need [this kind of workaround](https://www.npmjs.com/package/fast-glob#how-to-write-patterns-on-windows)?